resources/github-comment-markdown.template

```diff
@@ -73,15 +75,11 @@
   <p>
   ${(stepsErrors?.size() > 10) ? '> Show only the first 10 steps failures' : ''}
   <% stepsErrors?.take(10)?.each{ c -> %>
-  * **Name**: `${c?.displayName && c?.displayName != 'null' ? c?.displayName : ''}`
-    * Description: ${c?.displayDescription && c?.displayDescription != 'null' ? c?.displayDescription : ''}
-    <% if(c?.durationInMillis >= 0) {%>
-    * Duration: ${Math.round(c.durationInMillis/1000/60)} min ${Math.round(c.durationInMillis/1000)%60} sec
-    <%}%>
-    * Start Time: ${c?.startTime && c?.startTime != 'null' ? c?.startTime : ''}
-    <% if(c?.url && c?.url != 'null') {%>
-    * [log](${c?.url})
-    <%}%>
+  <% description = (c?.displayDescription && c?.displayDescription != 'null') ? "* Description: `${c?.displayDescription}`" : ''%>
+  <% duration = (c?.durationInMillis >= 0 ) ? "(took ${Math.round(c.durationInMillis/1000/60)} min ${Math.round(c.durationInMillis/1000)%60} sec)" : ''%>
+  <% url = (c?.url && c?.url != 'null') ? ". View log details on [here](${c?.url}/?start=0)" : ''%>
+  * **Name**: `${c?.displayName && c?.displayName != 'null' ? c?.displayName : ''}` ${duration} ${url}
```

Comment:
Aggregate the step name with the duration and logs URL, then we don't need three lines but one single line